model/src/test/kotlin/ProjectTest.kt

```diff
@@ -34,6 +34,35 @@ private fun readAnalyzerResult(analyzerResultFilename: String): Project =
         .readValue<ProjectAnalyzerResult>().project
 
 class ProjectTest : WordSpec({
+    "Projects" should {
+        "be sorted by type" {
+            val npmProject = Project.EMPTY.copy(id = Identifier.EMPTY.copy(type = "NPM"))
+            val mavenProject = Project.EMPTY.copy(id = Identifier.EMPTY.copy(type = "Maven"))
+
+            val projects = listOf(npmProject, mavenProject)
+
+            projects.sorted() shouldBe listOf(mavenProject, npmProject)
```

Comment:
Consider using `shouldContainExactly()` to make the test case not require the type `List` (without need)?
(Same below)

Reply:
Done.